pyproject.toml

```diff
@@ -67,6 +67,12 @@ test = [
     "pytest-timeout",
     "pytest-cov",
 ]
+ifeval = [
```

Comment:
Can this be moved to the [project.optional-dependencies] (like vllm above)?

Then, when you want to run an ifeval environment/need it's deps, you should run that ray remote with a PY_EXECUTABLE that points to that optional dep. Like the following for VLLM:

- specify environment with --extra ifeval
https://github.com/NVIDIA/NeMo-RL/blob/main/nemo_rl/distributed/virtual_cluster.py#L46

- launch worker with this py_executable
https://github.com/NVIDIA/NeMo-RL/blob/main/nemo_rl/models/generation/vllm.py#L52
(your sub-workers will also need it. See ifeval_environment L147)